airbyte-ci/connectors/live-tests/src/live_tests/conftest.py

```diff
@@ -642,5 +642,5 @@ def pytest_generate_tests(metafunc):
         requested_fixtures,
         [[c] * len(requested_fixtures) for c in all_connection_objects],
         indirect=requested_fixtures,
-        ids=[f"CONNECTION {hashlib.sha256(c.connection_id.encode()).hexdigest()[:7]}" for c in all_connection_objects],
+        ids=[f"CONNECTION {c.connection_id[:4]}" for c in all_connection_objects],
```

Comment:
Tobias is fine with the first 8 so if it can help reduce confusion, I would go for 8 characters